### PR TITLE
Use a cookie to detect if advanced filters pane should be open or not

### PR DESCRIPTION
**Pull request by punchagan**
- [x] Test plan | Unit test | Integration test
  1. Click on the advanced search button to open the advanced filters pane. 
  2. Reload the page to see that the advanced filters pane is open.
- [x] Copyright header
- [x] Code formatting
- [x] Documentation
- [x] Change log entry
```
Open the advanced filters pane if it was previously opened, not otherwise [#2741](https://github.com/akvo/akvo-rsr/issues/2741)
```

### CODE REVIEW

File: akvo/rsr/static/scripts-src/rsr.js [hunk 1]
```diff
@@ -38,6 +38,7 @@ $(document).ready(function() {
     e.preventDefault();
     $("#wrapper, #search").toggleClass("toggled");
     $("a.showFilters > i").toggleClass("fa-toggle-off fa-toggle-on");
+    Cookies.set(advancedFilterCookie, $("#wrapper, #search").hasClass('toggled')?'on':'off');
```
Comment (zzgvh) [inline akvo/rsr/static/scripts-src/rsr.js]: This splats, since advancedFilterCookie is undefined.

Comment (punchagan) [inline akvo/rsr/static/scripts-src/rsr.js]: Sorry for the broken PR. I think I had a local change, which I forgot to commit or something.

Comment (punchagan) [inline akvo/rsr/static/scripts-src/rsr.js]: This should be fixed now.

**Comment by punchagan** [comment]
This was a bug in the code, that didn't handle pages which didn't have the pane/sidebar. I fixed this now, and should work as expected. Thanks for the extensive testing, Gabriel!

**Pull request merged**